verification/curator-service/ui/src/components/LinelistTable.tsx

```diff
@@ -67,15 +67,18 @@ interface TableRow {
     admittedToHospital: string;
 }
 
-interface Props {
+interface LocationState {
+    newCaseIds: [string];
+    editedCaseIds: [string];
+}
+
+interface Props extends RouteComponentProps<any, any, LocationState> {
```

Comment:
Do you get a warning in VSCode for these `any`s? What about if you run `npm run lint`?

Reply:
Yes, I thought we were okay with ignoring warnings for small things like that. But in quick fixes I saw an option that said "Use 'never' instead, this is useful when instantiating generic type parameters that you don't need to know the type of." so switched to that instead.